### DISCUSSION TITLE
Documentación para Complemento de Comercio Exterior

### DISCUSSION THREAD

**Discussion by JuliusLancelot** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#61] [Q&A]
Hola, no encuentro documentación para generar el complemento de comercio exterior, me podrías ayudar con eso o con un ejemplo? Gracias!

**Comment by AndresRamos** [comment]
Hola Julio, necesito revisar como generar facturas con complemento. Ya teniendo mas información te agrego un ejemplo si es posible.
